src/data/set/intervals/unordered_interval.lean

```diff
@@ -0,0 +1,146 @@
+/-
+Copyright (c) 2020 Zhouhang Zhou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Zhouhang Zhou
+-/
+
+import data.set.intervals.disjoint
```

Comment:
It seems that you only need `basic`, not `disjoint`.

Reply:
One lemma mentions `bdd_below` and `bdd_above`.